sway/config/input.c

```diff
@@ -156,7 +158,7 @@ struct input_config *store_input_config(struct input_config *ic) {
 		merge_input_config(current, ic);
 		free_input_config(ic);
 		ic = current;
-	} else if (!wildcard) {
+	} else if (!wildcard && !type_wildcard) {
```

Comment:
This should be executed, otherwise the following will create four input configs for `type:touchpad`:
```
input type:touchpad {
  tap enabled
  tap_button_map lmr
  scroll_method edge
  natural_scroll disabled
}
```